Dockerfile

```diff
@@ -1,9 +1,11 @@
-FROM scratch
+FROM busybox
 
 MAINTAINER Tom Denham <tom@tigera.io>
 
 ADD dist/calico /opt/cni/bin/calico
 ADD dist/calico-ipam /opt/cni/bin/calico-ipam
+ADD k8s-install/scripts/install-cni.sh /install-cni.sh
```

Comment:
Is the point of this to install CNI to the host filesystem via the action of a  short-lived container?